Junk4Dollars-app/Auctions/Auction.swift

```diff
@@ -1,18 +1,32 @@
+import Foundation
+
 public struct Auction: Codable {
 
     public let identifier: Int
     public let title: String
+    public let description: String
+    public let starting_price: Int
+    public let ends_at_string: String
```

Comment:
I don't think it can be removed with the current way I'm having the Auction struct use codable to turn a date-string into a Date object. The only way to do that would be to specify how codable encodes the stringified json to the Auction structure, which seemed a lot more complicated.